controllers/v1/questionsController.js

```diff
@@ -255,19 +265,28 @@ module.exports = class Questions extends Abstract {
           let criteria = {};
           let ecm = {};
 
+          if (!parsedQuestion["evidenceMethod"]) {
+             parsedQuestion["evidenceMethod"] = solutionDocument.evidenceMethods[Object.keys(solutionDocument.evidenceMethods)[0]].externalId;
```

Comment:
@deepakr123 - This should only be for solution type survey

Reply:
resolved